views/newsletters.html

```diff
@@ -398,6 +398,7 @@
     'clickedLinks': __('Clicked Links'),
     'subscriberEngagement': __('Subscriber Engagement'),
     'googleAnalytics': __('GA campaign'),
+    'bounces': __('Bounces'),
```

Comment:
I think this needs a comment for the translator. Imagine translating it to the Czech without any context. You wouldn't know what bounces they talk about and you would probably translate it like a ball bounces (https://d.pr/i/faPubH). That would be incorrect and possibly confusing for the user. 

We need to explain that this is an email that bounced and what that means. I would add a context like: "Those are emails that couldn't be delivered for some reason: email doesn't exist, or the mailbox is full." Maybe ask PM for the context.

Reply:
I asked PM for the context.
Thanks for the catch!